### ISSUE TITLE
Difference between paper and code

### ISSUE THREAD

**Issue by valencebond** [Megvii-Nanjing/BBN#2]
<img width="245" alt="Screen Shot 2019-12-27 at 5 14 59 PM" src="https://user-images.githubusercontent.com/44313446/71511230-8a7b5780-28cc-11ea-8219-67f855e6e741.png">
the corresponding code is 

> mixed_feature = 2 * torch.cat((l * feature_a, (1 - l) * feature_b), dim=1)
        output = model(mixed_feature, classifier_flag=True)

according to the code, the target introduced in section 4.3 may can not achieve, as the feature is concatenated followed by only one classifer.

> Cumulative learning strategy is proposed to shift the learning focus between the bilateral branches by controlling the weights for features produced by two branches and the classiﬁcation loss L.

would you mind telling me the reason behind this change?

**Comment by ZhouBoyan** [comment]
Actually, two fully connected layer could be merged into one for simplicity.
Please refer to the formula below:
![cal](https://user-images.githubusercontent.com/58501117/71515073-6c1d5800-28dc-11ea-8d35-c33f63b096a0.jpg)

**Comment by valencebond** [comment]
thanks~ that means manifold mixup by concat is a better way than by originally sum ？what‘s more，why the scale 2 is needed？
> mixed_feature = 2 * torch.cat((l * feature_a, (1 - l) * feature_b), dim=1)

**Comment by ZhouBoyan** [comment]
Scale 2 is to ensure the grad consistent with the default combiner. (e.g. two samplers sample the same picture, and l = 0.5)

**Comment by nisargshah1999** [comment]
> thanks~ that means manifold mixup by concat is a better way than by originally sum ？what‘s more，why the scale 2 is needed？
> 
> > mixed_feature = 2 * torch.cat((l * feature_a, (1 - l) * feature_b), dim=1)

Hi @valencebond @ZhouBoyan Is this equivalent to that mentioned in the paper or concat method is better performing than original sum.

Thanks
